### PR TITLE
Update README.md Country wide 3D data links for The Netherlands

### PR DESCRIPTION
**Pull request by raymondnijssen**
Country wide 3D data links for The Netherlands

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ If you know about more interesting 3D data sources, please open issues with link
 -  Montréal (lod2 + textures)
    http://donnees.ville.montreal.qc.ca/dataset/maquette-numerique-batiments-citygml-lod2-avec-textures
 
-## Czech Republic
+## Czech
```
Comment (wonder-sk) [inline README.md]: "Czech" is an adjective - did you mean Czechia? :-)

Comment (raymondnijssen) [reply README.md]: Yea, that. Just remove the snobby "Republic" part!

**Comment by wonder-sk** [comment]
Thanks @raymondnijssen !

**Comment by raymondnijssen** [comment]
Welcome @wonder-sk !!

**Pull request merged**